sql_metadb/derived_tables/po_lines_cost

```diff
@@ -0,0 +1,41 @@
+-- These derived table extracts cost information on the purchase order line.
+
+DROP TABLE IF EXISTS po_lines_cost;
+
+CREATE TABLE po_lines_cost AS
+SELECT
+    pol.id AS pol_id,
+    jsonb_extract_path_text(pol.jsonb, 'cost', 'listUnitPrice')::numeric(19,4) AS po_line_list_unit_price_phys,
+    jsonb_extract_path_text(pol.jsonb, 'cost', 'quantityPhysical') AS po_lines_quant_phys,
```

Comment:
Misspelling of column 'po_lines_quant_phys' it should be 'po_line_quant_phys' to match the create index statement below.